flink-core/src/test/java/org/apache/flink/core/fs/local/LocalFileSystemTest.java

```diff
@@ -236,6 +237,7 @@ public void testRenameNonExistingFile() throws IOException {
     }
 
     @Test
+    @Ignore("FLINK-27082: Fails in Docker with root user")
```

Comment:
Wouldn't it be better to create interface something like `FailsOnDockerWithRoot`
and with help of `@Category` mark tests failing for the specific maven profile
like it is done for 
https://github.com/apache/flink/blob/cabb28d25c4c58af3ee23fc4a63f9564aefd6146/flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/testutils/junit/FailsOnJava17.java#L21

Reply:
That's actually a good idea. I haven't thought of it. But that way, we could disable the tests in GHA but keep them in Azure CI. I'm gonna look into it :+1:

Reply:
I updated the PR according to your idea. :+1: 

The following two GHA workflow runs include this PR:
* [build #68](https://github.com/XComp/flink/actions/runs/7292799860) does not have the profile activated
* [build #70](https://github.com/XComp/flink/actions/runs/7292859288) does have the profile activated

Reply:
Awesome, thanks for making it real :+1: